### DISCUSSION TITLE
indirect expansions and filter

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#1990] [Q&A] [answered]
Can I filter when i'm using indirect expand ? 
For example: `menu_item(menu).parent = ""`

**Comment by ganigeorgiev** [comment, marked as answer]
No, `filter` and `sort` for indirect relation fields at the moment is not supported.

It is in the [roadmap](https://github.com/orgs/pocketbase/projects/2) and eventually will be implemented in v0.15/v0.16.

**Reply by hungcrush** [reply]
Thank you. I will follow
